### DISCUSSION TITLE
Connection Popup

### DISCUSSION THREAD

**Discussion by ryanhowell18** [rustdesk/rustdesk#8579] [Q&A]
Good Evening,

I am wondering if anyone can help me, is it possible to hide the below pop up or stop it from showing completely when someone connects? 

![image](https://github.com/rustdesk/rustdesk/assets/144552706/cff96d46-c0b2-4a8c-b040-c434f820ee73)

Any helps is appreciated.

**Comment by rustdesk** [comment]
it is in rustdesk server pro version only.
